### PR TITLE
Feature witbook

### PR DESCRIPTION
**Pull request by LaurenSchenk1**
### Proposed changes
Documentation of WIT intro book and wetlands module

### Closes issues 
- Closes Issue #902 

### Checklist (replace `[ ]` with `[x]` to check off)
- [ x] Notebook created using the [DEA-notebooks template](https://github.com/GeoscienceAustralia/dea-notebooks/tree/develop)
- [x] Remove any unused Python packages from `Load packages`
- [x] Remove any unused/empty code cells
- [x] Remove any guidance cells (e.g. `General advice`)
- [x] Ensure that all code cells follow the [PEP8 standard](https://www.python.org/dev/peps/pep-0008/) for code. The `jupyterlab_code_formatter` tool can be used to format code cells to a consistent style: select each code cell, then click `Edit` and then one of the `Apply X Formatter` options (`YAPF` or `Black` are recommended).
- [x] Include relevant tags in the final notebook cell (refer to the [DEA Tags Index](https://docs.dea.ga.gov.au/genindex.html), and re-use tags if possible)
- [x] Clear all outputs, run notebook from start to finish, and save the notebook in the state where all cells have been sequentially evaluated
- [x] Test notebook on both the `NCI` and `DEA Sandbox` (flag if not working as part of PR and ask for help to solve if needed)
* hasn't been tested on NCI
- [x] If applicable, update the `Notebook currently compatible with the NCI|DEA Sandbox environment only` line below the notebook title to reflect the environments the notebook is compatible with

### CODE REVIEW

**Comment by robbibt** [comment]
Will have a look at this today, at first glance it's looking amazing @LaurenSchenk1 @BexDunn!

File: Tools/dea_tools/wetlands.py [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+    
+    '''
+       
+    # ignore high pixel missing timestamp result
+    polygon_base_df = polygon_base_df.dropna(subset=['bs'])
+    
+    # 1. compute the expected vegetation area total size: 1 - water (%) - wet (%)    
+    polygon_base_df.loc[:, "veg_areas"] = 1 - polygon_base_df['water'] - polygon_base_df['wet']
+    
+    # 2. normalse the vegetation values based on vegetation size (to handle FC values more than 100 issue)
```
Comment (jmettes) [inline Tools/dea_tools/wetlands.py]: s/normalse/normalise/

File: Tools/dea_tools/wetlands.py [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+    # convert the string to Python datetime format, easy to do display the result in PNG
+    polygon_base_df.loc[:, "date"] = pd.to_datetime(polygon_base_df["date"], infer_datetime_format=True)
+    
+    polygon_base_df.reset_index(inplace=True)
+    
+    return polygon_base_df
+
+def generate_low_quality_data_periods(df):
+    '''
+    This function generates low quality data periods, includimg the SLC off period: https://www.usgs.gov/faqs/what-landsat-7-etm-slc-data
```
Comment (jmettes) [inline Tools/dea_tools/wetlands.py]: s/includimg/including/

File: Tools/dea_tools/wetlands.py [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+    
+    # Add the first no-data times (SLC-off only)
+    LS5_8_gap_start = datetime.datetime(2011,11,1)
+    LS5_8_gap_end = datetime.datetime(2013,4,1)
+    
+    df.loc[df[(df['date'] >= LS5_8_gap_start) & (df['date'] <= LS5_8_gap_end)].index, "off_value"] = 100
+    
+    # periods with an observation density of less than four observations within a twelve month (365 days) period
+    for i in range(3, len(df) - 3):               
+        # can change to another threshold (like: 100 days) to test dyanmic no-data-period display
```
Comment (jmettes) [inline Tools/dea_tools/wetlands.py]: s/dyanmic/dynamic/

File: DEA_products/DEA_Wetlands_Insight_Tool.ipynb [hunk 4]
```diff
@@ -0,0 +1,2403 @@
+{
```
Comment (robbibt) [inline DEA_products/DEA_Wetlands_Insight_Tool.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    dc = datacube.Datacube(app="WITbook")
</pre><p>We usually use the exact name of the notebook for "app", e.g. </p><p> </p><p>app="DEA_Wetlands_Insight_Tool"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/1102/discussion/#comment-1276997104'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (robbibt) [inline DEA_products/DEA_Wetlands_Insight_Tool.ipynb:1]: <p>"wetland study area" or "wetland's study area"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/1102/discussion/#comment-1276997107'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (robbibt) [inline DEA_products/DEA_Wetlands_Insight_Tool.ipynb:1]: <p>Can we add a little more background to TCI here? Like what it is intended to do, roughly how it works? It's a pretty important part of this analysis, so would be good to have a nice introduction to it here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/1102/discussion/#comment-1276997112'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (robbibt) [inline DEA_products/DEA_Wetlands_Insight_Tool.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    # rasterise the shapefile where poly is the vector data and pv is the xarray template
</pre><p>Some comments start with capitals, some don't - would be nice to standardise (most other notebooks start comments with capitals)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/1102/discussion/#comment-1276997114'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

**Comment by robbibt** [comment]
This notebook looks awesome! Have made some small suggestions, but all super minor. 🙂 

Can we also check that `wetlands.py` has been Black formatted before we merge?

**Comment by BexDunn** [comment]
@robbibt @jmettes have addressed comments :)

**Review by robbibt — APPROVED** [review]
Looks fantastic, great work @LaurenSchenk1 and @BexDunn!

**Comment by robbibt** [comment]
Hey @BexDunn @LaurenSchenk1, are we ready to squash-and-merge this? Exciting!

**Review by BexDunn — APPROVED** [review]

**Comment by BexDunn** [comment]
Yep! All done! Was waiting for the tests to run last Friday, and have been off sick all week. How exciting! Well done @LaurenSchenk1

**Pull request merged**